flake.nix

```diff
@@ -1,29 +1,30 @@
- {
- description = "Home Manager configuration of arijit";
+{
+  description = "Home Manager configuration of arijit";
```

Comment:
Change the description to better explain the purpose of this Nix file. I had actually copied it from my own nix config and forgot to change the description.

Reply:
I removed the file. Should be fine if there's no description entry.